install_ffmpeg.sh

```diff
@@ -3,6 +3,13 @@
 set -ex
 
 ROOT="${1:-$HOME}"
+ARCH="$(uname -m)"
+
+if [[ $ARCH == "arm64" ]] && [[ $(uname) == "Darwin" ]]; then
+  # Detect Apple Silicon
+  IS_M1=1
+fi
+echo "Arch $ARCH Apple Silicon $IS_M1"
```

Comment:
WDYT of only printing the apple silicon bit when the variable is actually present? Otherwise it could get confusing when it is not an apple silicon.

```suggestion
echo "Arch $ARCH ${IS_M1:+(Apple Silicon)}"
```

Reply:
Thanks!
I'm bad at `bash`, so I've put a lot of prints to check that it works as I've intended.